tracer/src/Datadog.AutoInstrumentation.NativeLoader/CMakeLists.txt

```diff
@@ -194,7 +195,6 @@ if(ISLINUX)
             ${OUTPUT_DEPS_DIR}/fmt/src/fmt-build/libfmt.a
             -static-libgcc
             -static-libstdc++
```

Comment:
Can you please run an Azure DevOps build with [`use_native_loader` flag](https://github.com/DataDog/dd-trace-dotnet/blob/6f1635f2181159e96c2a83c1c217440b08ba8add/tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs#L47) enabled once (is it `false` by default) ?

Reply:
done here https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=93114&view=results

Reply:
Awesome. Looking good, other than the failures that are already fixed in #2551, I noticed one [WCF test failure](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=93114&view=ms.vss-test-web.build-test-results-tab&runId=5753722&resultId=100037&paneView=debug), but I think that's likely a flakey test that's unrelated to your PR. If you think that's the case, feel free to resolve 👍  Thanks!

Reply:
Yes, this test succeeded in a previous run.